automated install/basic-install.sh

```diff
@@ -233,19 +233,19 @@ if is_command apt-get ; then
         phpVer="php$phpInsMajor.$phpInsMinor"
     fi
     # We also need the correct version for `php-sqlite` (which differs across distros)
-    if ${PKG_MANAGER} install --dry-run ${phpVer}-sqlite3 > /dev/null 2>&1; then
+    if "${PKG_MANAGER}" install --dry-run "${phpVer}"-sqlite3 > /dev/null 2>&1; then
```

Comment:
The last argument can be combined into one string:
```
"${phpVer}-sqlite3"
```

Reply:
If that's the preferred style, I will apply it to all instances I find.

Reply:
Addressed.